config/v3_3_experimental/types/storage_test.go

```diff
@@ -39,10 +39,12 @@ func TestStorageValidate(t *testing.T) {
 			in: Storage{
 				Links: []Link{
 					{
-						Node: Node{Path: "/foo"},
+						Node:          Node{Path: "/foo"},
+						LinkEmbedded1: LinkEmbedded1{Target: util.StrToPtr("/foo-t")},
```

Comment:
Is the fact that this is required now due to the check being moved directly into the `Validate` function?